packages/mobile/src/screens/chat-screen/ChatBlastCTA.tsx

```diff
@@ -13,30 +12,26 @@ import {
   IconCaretRight
 } from '@audius/harmony-native'
 
+import { useAppDrawerNavigation } from '../app-drawer-screen'
+
 const messages = {
   title: 'Send a Message Blast',
   description: 'Send messages to your fans in bulk.',
   badgeRequired: 'Badge Required',
   or: 'or'
 }
 
-type ChatBlastCTAProps = {
-  onClick: () => void
-}
-
-export const ChatBlastCTA = (props: ChatBlastCTAProps) => {
-  const { onClick } = props
-
-  const { onOpen: openChatBlastModal } = useChatBlastModal()
-
+export const ChatBlastCTA = () => {
   const { data: userId } = useGetCurrentUserId({})
   const { tierNumber, isVerified } = useSelectTierInfo(userId ?? 0) ?? {}
-  const userMeetsRequirements = isVerified || (tierNumber && tierNumber > 0)
+  const userMeetsRequirements =
+    // DEV ONLY
+    isVerified || (tierNumber && tierNumber > 0) || true
```

Comment:
you should actually be able to use the remote config var which should come out to 0 for stage, lmk if it isn't. see this PR: https://github.com/AudiusProject/audius-protocol/pull/9490